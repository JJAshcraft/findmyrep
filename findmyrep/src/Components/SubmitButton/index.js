import styled from "styled-components";

const SubmitButton = styled.button`
  color: #fff;
  cursor: pointer;
  border: none;
  background-color: #06a7ed;
  width: 100%;
  border-radius: 2px;
  height: 40px;
  &:disabled {
    background-color: grey;
    cursor: not-allowed;
  }
`;

export default SubmitButton;
