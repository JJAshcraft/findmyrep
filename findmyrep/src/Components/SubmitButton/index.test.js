import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import SubmitButton from "./index.js";

test("HR component renders with proper styling", () => {
  const tree = renderer.create(<SubmitButton />).toJSON();
  expect(tree).toMatchSnapshot();
});
