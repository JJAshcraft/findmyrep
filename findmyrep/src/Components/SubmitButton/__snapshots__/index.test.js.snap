// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HR component renders with proper styling 1`] = `
.c0 {
  color: #fff;
  cursor: pointer;
  border: none;
  background-color: #06a7ed;
  width: 100%;
  border-radius: 2px;
  height: 40px;
}

.c0:disabled {
  background-color: grey;
  cursor: not-allowed;
}

<button
  className="c0"
/>
`;
